_projects/_template.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Example Astropy
+desc: This is the first idea for astropy
```

Comment:
What's the difference between this, and the "description" below?  I suggest removing this and having the name be the only "description" in this section